demo/rn/App.tsx

```diff
@@ -1,7 +1,10 @@
 import React from "react";
+import { LogBox } from "react-native";
 import { NavigationContainer } from "@react-navigation/native";
 import { RootNavigator } from "./src/screens/root-navigator";
 
+LogBox.ignoreLogs(["Require cycle: ../../packages/victory"]);
```

Comment:
Are we getting require cycles out of the box with victory-native? Or is this an artifact of the demo setup? (Just want to make sure end-users aren't suffering through require cycles, and if they are – put something on our roadmap to address that.)

Reply:
they're caused by upstream issues with `d3` that we can't easily address 😓 

more info and guidance [here](https://formidable.com/open-source/victory/docs/native#4-ignoring-require-cycles)